selendroid-server/build.gradle

```diff
@@ -0,0 +1,81 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:1.4.0-beta5'
+        classpath 'com.jakewharton.sdkmanager:gradle-plugin:0.12.0'
+    }
+}
+
+repositories {
+    jcenter()
+}
+
+apply plugin: 'android-sdk-manager'
+apply plugin: 'android'
+version = '0.18.0-SNAPSHOT'
+
+android {
+    compileSdkVersion 23
+    buildToolsVersion '23.0.1'
+
+    defaultConfig {
+        minSdkVersion 10
+        targetSdkVersion 16
+    }
+
+    lintOptions {
+        abortOnError false
+    }
+
+    sourceSets {
+        main {
+            manifest.srcFile 'AndroidManifest.xml'
+            res.srcDirs = ['res']
+            assets.srcDirs = ['assets']
+        }
+    }
+
+    packagingOptions {
+        exclude 'META-INF/INDEX.LIST'
+        exclude 'META-INF/io.netty.versions.properties'
+    }
+
+    buildTypes {
+        debug {
+            applicationVariants.all { variant ->
+                variant.outputs.each { output ->
+                    output.outputFile = new File(
+                            output.outputFile.parent,
+                            output.outputFile.name.replace('-debug.apk', "-${variant.versionName}.apk"))
+                }
+            }
+        }
+    }
+}
+
+configurations {
+    provided
+}
+
+sourceSets {
+    main.compileClasspath += configurations.provided
+    test.compileClasspath += configurations.provided
+    test.runtimeClasspath += configurations.provided
+}
+
+dependencies {
+    compile project(':selendroid-server-common')
+    compile 'org.json:json:20090211'
+    testCompile 'org.json:json:20090211'
+    testCompile 'junit:junit:4.12'
+    testCompile 'org.mockito:mockito-core:1.9.5'
+    testCompile 'commons-io:commons-io:2.2'
+    testCompile 'org.apache.httpcomponents:httpclient:4.2.1'
+    testCompile 'commons-logging:commons-logging:1.1.3'
+
+    provided files('../third-party/cordova-3.7.0/classes.jar')
+    provided files('../third-party/cordova-4.0.0/classes.jar')
+    provided files('../third-party/support-v4/classes.jar')
```

Comment:
this support classes.jar should be available in the android library in the sdk, we should be able to remove it from third-party (well, we can remove it when we remove the pom files)